assets/js/base/components/product-list/product-sort-select/index.tsx

```diff
@@ -54,7 +55,6 @@ const ProductSortSelect = ( { onChange, readOnly, value } ) => {
 					),
 				},
 			] }
-			readOnly={ readOnly }
```

Comment:
removed the readOnly property from the SortSelect component as it was never used, and was being applied to a <select> HTML Element which is not valid